packages/trip-form/src/TripOptions/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement, useState } from "react";
+
+import GeneralSettingsPanel from "../GeneralSettingsPanel";
+
+import FeaturedOption from "./FeaturedOption";
+import FeaturedOptionOverlay from "./FeaturedOptionOverlay";
+import ModeRow from "./ModeRow";
+import TransitOptions from "./TransitOptions";
+import { Company, QueryProps } from "./types";
+import * as S from "./styled";
+
+interface ComponentProps {
+  footer?: ReactElement;
+  supportedCompanies: Company[];
+}
+
+type Props = ComponentProps & QueryProps;
+
+export default function TripOptions(props: Props): ReactElement {
```

Comment:
Add JSDoc, and see the `location-icon` package for the export trick so that the component's JSDocs appear in storybook.

Reply:
Added in 6d0d62dd6cd2cba86e6fab24c6c8d7222ee0e239, but @landonreed could you possibly expand since I'm not entirely sure what some of them do